### PR TITLE
refactor(bigquery): update code samples of load table file and uri

### PR DESCRIPTION
**Pull request by HemangChothani**
Towards #8989 

### List of changes:

1. load_table_file
2. load_table_uri_avro
3. load_table_uri_csv
4. load_table_uri_json
5. load_table_uri_parquet
6. load_table_uri_orc
7. load_table_uri_cmek

### CODE REVIEW

File: bigquery/docs/usage/tables.rst [hunk 1]
```diff
@@ -110,6 +110,28 @@ Load a Parquet file from Cloud Storage:
 See also: `Loading Parquet data from Cloud Storage
 <https://cloud.google.com/bigquery/docs/loading-data-cloud-storage-parquet>`_.
 
+Load a Avro file from Cloud Storage:
```
Comment (shollyman) [inline bigquery/docs/usage/tables.rst]: s/a Avro/an Avro/

File: bigquery/docs/usage/tables.rst [hunk 2]
```diff
@@ -110,6 +110,28 @@ Load a Parquet file from Cloud Storage:
...
+.. literalinclude:: ../samples/load_table_uri_avro.py
+   :language: python
+   :dedent: 4
+   :start-after: [START bigquery_load_table_gcs_avro]
+   :end-before: [END bigquery_load_table_gcs_avro]
+
+See also: `Loading Avro data from Cloud Storage
+<https://cloud.google.com/bigquery/docs/loading-data-cloud-storage-avro>`_.
+
+Load a ORC file from Cloud Storage:
```
Comment (shollyman) [inline bigquery/docs/usage/tables.rst]: s/a Orc/an Orc/

File: bigquery/samples/load_table_file.py [hunk 3]
```diff
@@ -0,0 +1,43 @@
+# Copyright 2019 Google LLC
```
Comment (shollyman) [inline bigquery/samples/load_table_file.py]: Please update copyright notices to 2020 in the new files

**Comment by HemangChothani** [comment]
`test_client_load_partitioned_table` snippet failed is unrelated to changes.

**Comment by plamut** [comment]
Merging, as the PR has been approved, and the comments in the approval have been addressed.

**Pull request merged**